docs/spec_draft.md

```diff
@@ -3731,6 +3819,50 @@ where `pred_val = rank(pred) == 0 ? pred : pred[i0, ..., iR-1]`.
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.send
+
+### Semantics
+
+Sends data `inputs` from one partition to a [stablehlo.recv](#stablehlorecv)
+instruction, in another partition, that shares the same channel handle pair
+{`channel_id`, `channel_type`}.
+
+If `is_host_transfer`, an optional input with default value as `fasle`, is
+`true`, then the operation transfers data to the host.
```

Comment:
Proposed rewording:

```
Sends `inputs` to a channel `channel_id`.

The operation takes a `token` and produces a `token` to reify the side effect
of this operation as a value that other operations can take a data dependency on.

If `is_host_transfer` is `true`, then the operation transfers data to the
host. Otherwise, it transfers data to another device. What this means is
implementation-defined.
```

Same feedback as above.